src/components/Button/Button.tsx

```diff
@@ -36,12 +11,10 @@ interface Props {
 
 const Button = (props: Props) => {
    return (
-      <StyledButton color={props.color} size={props.size}>
+      <SButton color={props.color} size={props.size} icon={props.icon}>
          {props.text}
-         <span>
-            {props.icon ? <FontAwesomeIcon icon={icons[props.icon]} /> : ""}
-         </span>
-      </StyledButton>
+         {props.icon && <Sicon className="icon" icon={props.icon} />}

```

Comment:
1. try keeping the convention of upper case in the beginning of the word Sicon => SIcon
2. why we need the className if we have the style element?

Reply:
1.done
2. I need to select that element in the parent's styled component for the hover effect, if you have another suggestion I'm open for suggestions.